app/assets/stylesheets/views/_homepage.scss

```diff
@@ -159,3 +159,48 @@
   @include govuk-font(36, $weight: bold);
   color: govuk-colour("white");
 }
+
+.homepage-links-and-search {
+  background: govuk-colour("light-grey");
+  padding-bottom: govuk-spacing(8);
+  padding-top: govuk-spacing(6);
+
+  @include govuk-media-query($from: "desktop") {
+    padding-bottom: govuk-spacing(8);
```

Comment:
Redundant spacing as it's already specified in the mobile styling on line 165.

Reply:
Removed in https://github.com/alphagov/frontend/pull/3028/commits/53a013dac4444d551a5682b4e30278f437f79f44.